### PR TITLE
Update live page

### LINKED ISSUE #9: Update Live Page
- [x] update text for 2018
- [x] add current partners
- [x] add challenges?
- [x] add mentor info
- [x] ...
- [x] add Twilio code (see https://trello.com/c/bxeg9JCX/632-twilio-promocodes-anfragen)
- [ ] extend open data source

### PR DESCRIPTION
**Pull request by jonashering**
closes #9 

TODO:
- [x] wait for updated schedule and include it (especially workshops)
- [ ] check sap challenges
- [x] check wwf challenges
-  [x] check if hardware list is up-to-date

Can do later:
- [x] add mentors availabilityn (doodle on trello card)
- [x] replace mentor emails with Slack link
- [x] check Paul (mentor) is **not** on the page
- [ ] check if all TBAs are filled (e.g. wifi pw)
- [ ] wait for prizes (main/sponsors --> all) and include them
- [ ] add more open apis

@MasterCarl @jak-ing @FWirtz

### CODE REVIEW

**Comment by jak-ing** [comment]
I think we shouldn't put the mentor's email addresses on our website in clear text... Rather put their Slack handle as text and as href a link which takes you straight to their Slack profile.

**Comment by FWirtz** [comment]
@jak-ing agree!
try this one: slack://user?team=T7ZDTGLU9&id=U804J2JTE
It should open a direct message with Lukas. Requires the slack app though.

The URL is build as follows:
```
slack://user?team=T7ZDTGLU9&id={member ID}
```
You can get the member id by clicking on the people in slack and then clicking on the down arrow next to `Call` and `Message` buttons on the right.

**Comment by lheilmann** [comment]
Here is a list of the Slack names of the mentors: 

screen shot 2018-05-21 at 14 16 06

I agree with @FWirtz that there should be a direct link to a chat with the particular mentor.

**Comment by lheilmann** [comment]
Please be aware of mentors who do not want to have their name on the live page 👍

**Comment by FWirtz** [comment]
@lheilmann are there any?

**Comment by lheilmann** [comment]
Paul does not want to have his name on the live page, see https://docs.google.com/forms/d/154Ty3gGeRy3wKI-6hnlfUqE5eOEyQjd3gD1vy41M2SI/edit#responses

**Comment by jak-ing** [comment]
@lheilmann just removed the link you posted. This repo is public and the link you posted is public as well! Please be more careful with this...

**Comment by jak-ing** [comment]
Updated Tableau and SAP challenges. 
Wording for SAP's challenge #2 is not final yet, they want to include more details.

**Comment by FWirtz** [comment]
Suggestion: lets remove the mentor "pictures"/placeholders

Also, the WWF and UFZ logos may definitely not be used like this on the dark background.

File: live/index.html [hunk 1]
```diff
@@ -0,0 +1,747 @@
...
+						<p>If your university uses EduRoam you can use that as well.</p>
+					</div>
+					<div class="faq-question">
+						<h3 class="text-orange">What if I don't have a team?</h3>
+						<p>Don't worry! At the end of our opening we will have a "match making" where you can find each other based on the idea
+							you want to work on. We will explain it in detail on Saturday.</p>
+
+						<p>Or you can use the  <a href="https://hackhpi2018.slack.com/messages/matchmaking" target="_blank">#matchmaking</a> channel in our Slack channel already to pitch your idea and find team mates&nbsp;:)</p>
+
+						<p>Teams should have a size of not more than 3-4.</p>
```
Comment (jak-ing) [inline live/index.html]: According to current discussion in #internal, please make this more firm, e.g. "maximum team size is 4, we encourage teams of 3-4" and maybe create an own question for it. fyi @FWirtz

File: live/index.html [hunk 2]
```diff
@@ -0,0 +1,747 @@
...
+					<p>The climate crisis is no far off phenomena, it has reached our homes. Its consequences, for example extreme weather
+						events like heavy rain and heatwaves, is affecting the nature just outside our front doors. Growth patterns of plants
+						and crops change just like behavioural patterns of animals such as migratory birds. Some plants and animals cannot
+						cope with the changing environment, they disappear altogether.</p>
+
+					<p>We want people not only to open their eyes to these changes. We want them to share their observations with others, building
+						a network of locally engaged citizens who connect over the discussion of climate change in their region. Ideally, we
+						want them to become active in the fight against climate change.</p>
+
+					<h3>#2 How can we make climate change more tangible? pt. 2</h3>
```
Comment (lheilmann) [inline live/index.html]: This challenge is apparently wrong. As you can see both challenges have the same title, meaning that there are two options provided by WWF for the first challenge. I do not know which one to take, but @annkatrinkuessner should know. The second challenge (#2) is found in the known shared document.

Comment (jonashering) [reply live/index.html]: right, back then the 2nd was not yet in the doc; is updated now

for challenge #1, both options are included now. can you confirm this is correct? @annkatrinkuessner

**Comment by jonashering** [comment]
Wenn die SAP challenge approved ist, würde ich schon mergen, damit sich alle Teilnehmer die Challenges anschauen können. Open APIs und die anderen Sachen können wir dann noch vor Samstag hinzufügen.

@jak-ing @FWirtz @lheilmann @annkatrinkuessner

**Pull request merged**